packages/hardhat-foundry/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@nomicfoundation/hardhat-foundry",
+  "version": "1.0.0",
+  "description": "Hardhat plugin that adds Hardhat support to Foundry projects",
+  "repository": "github:nomicfoundation/hardhat",
+  "homepage": "https://github.com/nomicfoundation/hardhat/tree/main/packages/hardhat-foundry",
+  "author": "Nomic Foundation",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "hardhat-plugin",
+    "foundry"
+  ],
+  "scripts": {
+    "lint": "yarn prettier --check && yarn eslint",
+    "lint:fix": "yarn prettier --write && yarn eslint --fix",
+    "eslint": "eslint 'src/**/*.ts'",
+    "prettier": "prettier \"**/*.{js,md,json}\"",
+    "test": "mocha --recursive \"test/**/*.ts\" --exit",
+    "build": "tsc --build .",
+    "prepublishOnly": "yarn build",
+    "clean": "rimraf dist"
+  },
+  "files": [
+    "dist/src/",
+    "src/",
+    "LICENSE",
+    "README.md"
+  ],
+  "dependencies": {
+    "chalk": "^2.4.2"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.0",
+    "@types/mocha": "^9.1.0",
+    "@types/node": "^14.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "4.29.2",
+    "chai": "^4.2.0",
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-import": "2.24.1",
+    "eslint-plugin-prettier": "3.4.0",
+    "hardhat": "^2.11.0",
+    "mocha": "^10.0.0",
+    "prettier": "2.4.1",
+    "rimraf": "^3.0.2",
+    "ts-node": "^8.1.0",
+    "typescript": "~4.5.2"
+  },
+  "peerDependencies": {
+    "hardhat": "^2.12.3"
```

Comment:
Reminder: this needs to be the latest version, the one that it's published with this PR